pkg/vm/opcode/from_string.go

```diff
@@ -0,0 +1,20 @@
+package opcode
+
+import "errors"
+
+var stringToOpcode = make(map[string]Opcode)
+
+func init() {
+	for i := 0; i < 255; i++ {
+		op := Opcode(i)
+		stringToOpcode[op.String()] = op
```

Comment:
There are some invalid opcodes and you're generating an `Opcode(XXX)` map for them, I don't think that's correct.

Reply:
I thought about this and see no big problem here: the string must be precisely defined (we don't return opcode for any random string), it is nice to have `FromString(op.String()) == op` as well as `FromString(s).String() == s`, and if really needed, it is better to have a separate function to check if opcode is implemented by our VM.